src/client/netsblox-dev.html

```diff
@@ -25,6 +25,10 @@
         <script type="text/javascript" src="Snap--Build-Your-Own-Blocks/FileSaver.min.js"></script>
 
         <!-- Snap overrides -->
+        <script type="text/javascript" >
+            const SERVER_ADDRESS = 'https://editor.netsblox.org',
```

Comment:
We should not to mix deployment configuration information into the code base (though I know there is one in the server CORS checking atm). This could be configured by an environment variable (added to the html template by the server) with the netsblox deployment as a default though